File: app/src/main/java/com/geowatershed/app/data/model/SiteIndicator.kt
package com.geowatershed.app.data.model

/**
 * A single site indicator row. [value] is null when the data source has no
 * record for this parcel yet — the "honest gap" state — never a placeholder
 * zero or an error.
 */
data class SiteIndicator(
    val name: String,
    val sub: String,
    val value: String?,
) {
    val isAvailable: Boolean get() = value != null
}

File: geowatershed_flutter/lib/models/site_indicator.dart
/// A single site indicator row.
///
/// [value] is null when the data source has no record for this parcel — the
/// honest gap. Never a placeholder zero, never an error, and never counted as
/// if it lowered risk.
class SiteIndicator {
  const SiteIndicator({required this.name, required this.sub, this.value});

  final String name;
  final String sub;
  final String? value;

  bool get isAvailable => value != null;
}
